### PR TITLE
BI-8: TamHuynh - Week 5: Develop test script for below functions o…

### PR DESCRIPTION
**Pull request by tamhuynhanh**
## Exercise

* [BL8-Develop test script for below functions of Opencart web application ](https://github.com/NT-Line-team/Behave_test/issues/8)
**##Overview**
Develop test script for below function of Opencart web application:
Login
1. Execute a successful login with an user.
2. Register
3. Forgot password
Test the forgot password function to make sure that the user receive an inform message after submitting the request.
**Verification Steps:**
Plz run below commands:
cd ./Behave_test/exercises/tamhuynh-selenium-week5
To execute login test script plz run python ./login.py
To execute register test script plz run python ./register.py
To execute forgot password test script plz run python ./forgot_password.py

### CODE REVIEW

**Comment by duynguyenx** [comment]
Hi @tamhuynhanh . Pls see below my comments for your pull request
1. Pls follow this doc https://github.com/NT-Line-team/Behave_test/wiki/Rules-and-Conventions to correct yours:
a. Summary 
b. Description
c. Attribute 
d. Tags.
e. Format
2. Change your commit message, it should be the short explanation about what you did?
3. Pls put your test script to Behave_test/exercises/<yourbranchname>/<example_script_name.py>
4. Missing test script for Forgot password function.
For reference, pls open this PR:
https://github.com/NT-Line-team/Behave_test/pull/57

File: bai_tap_week5/register.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+driver.find_element_by_id('input-address-1').send_keys('cong hoa')
+driver.find_element_by_id('input-city').send_keys('HCM')
+driver.find_element_by_id('input-postcode').send_keys('0084')
+driver.find_element_by_id('input-country').send_keys('Viet Nam')
+driver.find_element_by_id('input-zone').send_keys('Ho Chi Minh City')
+driver.find_element_by_id('input-password').send_keys('12345')
+driver.find_element_by_id('input-confirm').send_keys('12345')
+driver.find_element_by_xpath("""//*[@id="content"]/form/div/div/input[1]""").click()
+driver.find_element_by_xpath("""//*[@id="content"]/form/div/div/input[2]""").click()
+driver.implicitly_wait(20)
```
Comment (duynguyenx) [inline bai_tap_week5/register.py]: Pls remove this line `driver.implicitly_wait(20) `

**Comment by tamhuynhanh** [comment]
hi @hungphamviet  @duynguyenx ,
I rechecked my pull request, plz help me check again, 

Thank you

**Comment by duynguyenx** [comment]
Good effort @tamhuynhanh, It'd be better if you can give some headline for each section, pls see example below
 
## Exercise
* [Exercise- Week 5](https://github.com/NT-Line-team/Behave_test/issues/8)

## Overview
Develop test script for below function of Opencart web application:
1. Login
Execute a successful login with admin user.
2. Register
Register a new user.
3. Forgot password
Test the forgot password function to make sure that the user receive an inform message after submitting the request.

## Verification Steps:
Pls run below commands:
1. `cd /Behave_test/exercises/tamhuynh`
2. To execute login test script pls run `python ./login.py`
3. To execute register test script pls run `python ./register.py`
4. To execute forgot password test script pls run `python ./forgot_password.py`

**Comment by tamhuynhanh** [comment]
@duynguyenx  I changed, A Duy please help me review, thank you

File: exercises/tamhuynh-selenium-week5/register.py [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+driver.find_element_by_id('input-address-1').send_keys('cong hoa')
+driver.find_element_by_id('input-city').send_keys('HCM')
+driver.find_element_by_id('input-postcode').send_keys('0084')
+driver.find_element_by_id('input-country').send_keys('Viet Nam')
+driver.find_element_by_id('input-zone').send_keys('Ho Chi Minh City')
+driver.find_element_by_id('input-password').send_keys('12345')
+driver.find_element_by_id('input-confirm').send_keys('12345')
+driver.find_element_by_xpath("""//*[@id="content"]/form/div/div/input[1]""").click()
+driver.find_element_by_xpath("""//*[@id="content"]/form/div/div/input[2]""").click()
+driver.implicitly_wait(20)
```
Comment (duynguyenx) [inline exercises/tamhuynh-selenium-week5/register.py]: `implicitly_wait` should be put after line 10 `driver.maximize_window()`. We will have a topic for Wait in Week 6, @hungphamviet will give u guys some knowledge of Wait methods in Selenium library.

File: python_basic/tamhuynh/Print.py [hunk 3]
```diff
@@ -0,0 +1,21 @@
+__author__ = '8.1'
```
Comment (duynguyenx) [inline python_basic/tamhuynh/Print.py]: Pls remove this file

File: exercises/tamhuynh-selenium-week5/forgot_password.py [hunk 4]
```diff
@@ -0,0 +1,13 @@
+__author__ = '8.1'
```
Comment (duynguyenx) [inline exercises/tamhuynh-selenium-week5/forgot_password.py]: All exercises will be put in the same folder and it should be your name, pls correct
for example:
`exercises/tamhuynh/login.py`

**Pull request closed without merging**